### DISCUSSION TITLE
Copying preferences across systems/versions?

### DISCUSSION THREAD

**Discussion by manojkarthick** [rxhanson/Rectangle#431] [Q&A] [answered]
Thanks a lot for creating this awesome project! I followed the steps in the [README](https://github.com/rxhanson/Rectangle#preferences-storage) to import/export shortcuts across systems by transferring the plist file at `~/Library/Preferences/com.knollsoft.Rectangle.plist`. But Rectangle fails to pick up these values.

* Source Machine: macOS Catalina 10.15.6, Rectangle 0.43 (48)
* Target Machine: macOS Big Sur 11.2.2, Rectangle 0.44 (49)

Has the format changed for v0.44 or is transferring across different version unsupported?

My Preferences plist file for reference: [com.knollsoft.Rectangle.plist](https://github.com/manojkarthick/dotfiles/blob/master/config/rectangle/com.knollsoft.Rectangle.plist)

**Comment by rxhanson** [comment, marked as answer]
This should work fine. You do have to restart Rectangle at the destination for it to pick up the values. Sometimes users have mistakenly copied their plist to the wrong directory, so it's worth double checking to make sure it's in the right place. MacOS will cache values, so if you've done everything properly and it still isn't working, you can force the defaults cache to refresh with this command:

killall -u $USER cfprefsd

In v0.44, buttons were added to the settings tab of the preferences window to allow export/import of your config as json files. It might be worth it to just update your source machine to 0.44, export as json, and move it over to your destination machine.

**Reply by manojkarthick** [reply]
Thanks a lot @rxhanson - I updated both machines to v0.44 and used the import/export button to share my settings. I also opened a pull request to add this information to the README.md as I thought it would be useful for people who did not notice it in the changelog. https://github.com/rxhanson/Rectangle/pull/432
